amazon_msk/assets/dashboards/overview.json

```diff
@@ -1,429 +1,743 @@
+                "font_size": "18",
+                "has_padding": true,
+                "show_tick": false,
+                "text_align": "left",
+                "tick_edge": "left",
+                "tick_pos": "50%",
+                "type": "note"
             },
+            "id": 4560547737574062,
             "layout": {
-                "x": 115,
-                "y": 8,
-                "width": 47,
-                "height": 15
+                "height": 2,
+                "width": 8,
+                "x": 0,
+                "y": 0
             }
         },
         {
-            "id": 1,
             "definition": {
-                "type": "query_value",
-                "requests": [
-                    {
-                        "q": "avg:aws.kafka.offline_partitions_count{*}",
-                        "aggregator": "avg",
-                        "conditional_formats": [
-                            {
-                                "comparator": ">",
-                                "value": 1,
-                                "palette": "white_on_red"
-                            },
-                            {
-                                "comparator": "<=",
-                                "value": 0,
-                                "palette": "white_on_green"
-                            }
-                        ]
-                    }
+                "check": "aws.msk.can_connect",
+                "group_by": [
+                    "$cluster_arn",
+                    "$region_name"
                 ],
-                "custom_links": [],
-                "title": "# Offline",
-                "title_size": "16",
+                "grouping": "cluster",
+                "tags": [
+                    "*"
+                ],
+                "title": "Brokers",
                 "title_align": "left",
-                "precision": 0
+                "title_size": "16",
+                "type": "check_status"
             },
+            "id": 2169915587455394,
             "layout": {
-                "x": 1,
-                "y": 20,
-                "width": 17,
-                "height": 10
+                "height": 2,
+                "width": 4,
+                "x": 8,
+                "y": 0
             }
         },
         {
-            "id": 2,
             "definition": {
-                "type": "query_value",
+                "background_color": "vivid_blue",
+                "layout_type": "ordered",
+                "show_title": true,
+                "title": "Partitions",
+                "type": "group",
+                "widgets": []
```

Comment:
Shoudn't the widgets be inside the group?

Reply:
TIL, ty